src/main/scala/beam/router/LinkTravelTimeContainer.scala

```diff
@@ -10,64 +10,65 @@ import org.matsim.api.core.v01.population.Person
 import org.matsim.core.router.util.TravelTime
 import org.matsim.vehicles.Vehicle
 
-class LinkTravelTimeContainer(fileName: String, timeBinSizeInSeconds: Int) extends TravelTime with LazyLogging {
+import scala.collection.mutable
 
-  private var linkTravelTimeMap: Map[Id[Link], Map[Int, Double]] = Map()
+class LinkTravelTimeContainer(fileName: String, timeBinSizeInSeconds: Int, maxHour: Int = 24) extends TravelTime with LazyLogging {
```

Comment:
@wrashid , @colinsheppard Is it ok to have this parameter `maxHour` to pre-allocate array? I can make it configurable.

Reply:
@REASY does maxHour imply that no travel time data will have a tick corresponding to any hour >24? Or do the travel time bins wrap around (i.e. get modded by 24)?

@wrashid please confirm desired behavior, but I would imagine that we want the bins to wrap around, and therefore this pre-allocation approach should work. It will, however, become a problem if users want to use BEAM in the context of multi-day simulations and they expect the network travel times to contain bins beyond 24 hours.

Reply:
@colinsheppard , `maxHour` here is used as the size of array to be allocated. What I've seen in `0.linkstats.csv` is that it contains hours from 0 to 23 for per link and it seems to me that there is no skip hours. Basically this will be more efficient by memory to store in `Array` instead of `Map`.  If we have more than 24 hours in simulation, we can just make this parameter configurable